### PR TITLE
Change project keys

### PR DESCRIPTION
**Pull request by pratid**
#### Reviewers
@flopezag 

#### Description
Adoption of IoT Platform's project key pattern

### CODE REVIEW

File: ngsi_adapter/script/build/jenkins.sh [hunk 1]
```diff
@@ -93,7 +93,7 @@ PRODUCT_RELEASE=$(echo "$PRODUCT_INFO" | sed -n '/"release"/ {s/.*:.*"\(.*\)".*/
 PROJECT_NAME=$(sed -n '/"name"/ {s/.*:.*"\(.*\)".*/\1/; p; q}' $PROJECT_BASE_DIR/package.json)
 PROJECT_VERSION=$(sed -n '/"version"/ {s/.*:.*"\(.*\)".*/\1/; p; q}' $PROJECT_BASE_DIR/package.json)
 SONAR_PROJECT_NAME=$(echo "$PRODUCT_NAME-$PROJECT_NAME" | tr '_' '-')
-SONAR_PROJECT_KEY=com.telefonica.fiware:$SONAR_PROJECT_NAME
+SONAR_PROJECT_KEY=com.telefonica.IoT:$SONAR_PROJECT_NAME
```
Comment (flopezag) [inline ngsi_adapter/script/build/jenkins.sh]: sorry:

com.telefonica.iot

File: ngsi_event_broker/script/build/jenkins.sh [hunk 2]
```diff
@@ -92,7 +92,7 @@ PRODUCT_RELEASE=$(sed -n '/\[PRODUCT_RELEASE\]/ {s/.*\[.*\].*\[\(.*\)\].*/\1/; p
 PROJECT_NAME=$(sed -n '/AC_INIT/ {s/.*\[PRODUCT_NAME-\(.*\)].*/\1/; p}' $PROJECT_DIR/configure.ac)
 PROJECT_VERSION=$(sed -n '/AC_INIT/ {s/.*,[ \t]*\(.*\))/\1/; p}' $PROJECT_DIR/configure.ac)
 SONAR_PROJECT_NAME=$(echo "$PRODUCT_NAME-$PROJECT_NAME" | tr '_' '-')
-SONAR_PROJECT_KEY=com.telefonica.fiware:$SONAR_PROJECT_NAME
+SONAR_PROJECT_KEY=com.telefonica.IoT:$SONAR_PROJECT_NAME
```
Comment (flopezag) [inline ngsi_event_broker/script/build/jenkins.sh]: sorry:

com.telefonica.iot

**Comment by flopezag** [comment]
la key del sonar debería ser:

com.telefonica.iot:monitoring-ngsi-adapter
com.telefonica.iot:monitoring-ngsi-eventbroker

y el sonar project name debería ser:

Monitoring NGSI Adapter
Monitoring NGSI Event Broker

**Comment by pratid** [comment]
Event Broker are two separate words. Do you really mean "ngsi-eventbroker" in the Sonar key?

**Comment by pratid** [comment]
Another question: the Linux packages for these two components are named:

fiware-monitoring-ngsi-adapter_{version}_{arch}.deb
fiware-monitoring-ngsi-event-broker_{version}_{arch}.deb

I find very useful and convenient to prepend "fiware-" to those names. Should we change prefix to "iot-"?

**Comment by flopezag** [comment]
1) PaaSManager are also 2 words and we decided to put together... It doesn't matter what is the name but internally we put it. If we change it now we should change in other place too. If you are happy with two separate words well say me that you adopt it.

2) Why do you say that fiware-* is very useful and convenient, for my point of view it is redundant information that we should resolve, shouldn't?

**Comment by flopezag** [comment]
LGTM

**Pull request merged**